sootup.callgraph/src/main/java/sootup/callgraph/GraphBasedCallGraph.java

```diff
@@ -23,10 +23,11 @@
  */
 
 import com.google.common.base.Preconditions;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Set;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
```

Comment:
Change import java.util.*; back to 
import java.util.Comparator;
import java.util.HashMap;
import java.util.Map;
import java.util.Set;